### PR TITLE
enable ability to define columns by default table by table

### PR DESCRIPTION
**Pull request by EeroAnt**
small changes to multiple files. Column 'Id' was hidden by default on every table and this was set in tableView.tsx. Atm it looks like a lot of copy-paste code, but it enables to define which columns are shown/hidden by default on a table by table basis. This is related to #241

### CODE REVIEW

File: frontend/src/components/TableView/TableView.tsx [hunk 1]
```diff
@@ -40,6 +42,7 @@ export const TableView = <T extends MRT_RowData>({
   title?: string
   data: T[] | undefined
   columns: MRT_ColumnDef<T>[]
+  hiddenColumns: MRT_VisibilityState
```
Comment (iritmaximus) [inline frontend/src/components/TableView/TableView.tsx:45]: Should this be actually renamed to visibleColumns or similar. The logic is the "wrong" way around: `hiddenColumns = { id: false }` and not `id: true`.

File: frontend/src/components/TableView/TableView.tsx [hunk 2]
```diff
@@ -92,7 +95,7 @@ export const TableView = <T extends MRT_RowData>({
       density: 'compact',
     },
     initialState: {
-      columnVisibility: { id: false },
+      columnVisibility: hiddenColumns, // HERE ERalfnkJAKFÖLCN!!!!!!!!!!!!!!!
```
Comment (iritmaximus) [inline frontend/src/components/TableView/TableView.tsx:98]: Also what :D

**Review by iritmaximus — CHANGES_REQUESTED** [review]

File: frontend/src/components/TableView/TableView.tsx (hunk 2 above)
Comment (EeroAnt) [reply frontend/src/components/TableView/TableView.tsx:98]: :D I made mark that I found where the magic happens. Didn't want to lose it

File: frontend/src/components/TableView/TableView.tsx (hunk 1 above)
Comment (EeroAnt) [reply frontend/src/components/TableView/TableView.tsx:45]: Makes sense. Will do

**Review by iritmaximus — APPROVED** [review]
LGTM

**Pull request merged**